### PR TITLE
TS-Style fix for EditScheduleTable and BusyRooms

### PR DESCRIPTION
**Pull request by HiroFive**

### CODE REVIEW

File: frontend/src/components/Auth/Auth.scss [hunk 1]
```diff
@@ -57,28 +52,22 @@
...
         }
-
-        .form-field {
-            width: 25em;
-        }
     }
     .auth-card-header {
-        .title {
+        .auth-card-title {
             margin: {
```
Comment (ViktoriiaKosovan) [inline frontend/src/components/Auth/Auth.scss:61]: use shorthand property

File: frontend/src/components/Navigation/NavigationPanel.scss [hunk 2]
```diff
@@ -31,10 +32,10 @@
         justify-content: center;
         padding: 0;
         .navigation-select {
-            min-width: 100%;
-            color: $white;
             display: flex;
             height: 40px;
+            min-width: 100%;
```
Comment (Niknikson) [inline frontend/src/components/Navigation/NavigationPanel.scss]: $full-size

**Review by Niknikson — APPROVED** [review]

File: frontend/src/components/EditCurrentSchedule/ScheduleBoard/ScheduleBoard.scss [hunk 3]
```diff
@@ -82,8 +82,9 @@
     }
```
Comment (ViktoriiaKosovan) [inline frontend/src/components/EditCurrentSchedule/ScheduleBoard/ScheduleBoard.scss:80]: in row 74 use mixin for flex

File: frontend/src/components/Navigation/NavigationPanel.scss (hunk 2 above)
Comment (ViktoriiaKosovan) [inline frontend/src/components/Navigation/NavigationPanel.scss]: use constants

**Review by ViktoriiaKosovan — APPROVED** [review]

File: frontend/src/components/Navigation/NavigationPanel.scss (hunk 2 above)
Comment (HiroFive) [reply frontend/src/components/Navigation/NavigationPanel.scss]: done

File: frontend/src/components/EditCurrentSchedule/ScheduleBoard/ScheduleBoard.scss (hunk 3 above)
Comment (HiroFive) [reply frontend/src/components/EditCurrentSchedule/ScheduleBoard/ScheduleBoard.scss:80]: fixed

File: frontend/src/components/Navigation/NavigationPanel.scss (hunk 2 above)
Comment (HiroFive) [reply frontend/src/components/Navigation/NavigationPanel.scss]: changed

**Review by Yashchemskyi-Oleksandr — APPROVED** [review]

**Pull request merged**